staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -73,9 +75,13 @@ func SchemaDeclType(s *schema.Structural) *DeclType {
 					enumValues = append(enumValues, e.Object)
 				}
 			}
-			if fieldType := SchemaDeclType(&prop); fieldType != nil {
-				fields[Escape(name)] = &DeclField{
-					Name:         Escape(name),
+			if fieldType := SchemaDeclType(&prop, prop.XEmbeddedResource); fieldType != nil {
+				propName, ok := Escape(name)
+				if !ok {
+					propName = name // we retain the property for schema integrity even though it is impossible to reference in go programs
```

Comment:
Yes, you're right, since this is only for the CEL declaration, I can skip it.  I still have the schema information for when I do listMap equality checking (which is why I was being so cautious with the handling of unescapable key fields).

Reply:
Removed. The existing "listMaps with unsupported identity characters in property names" test verifies that equality still works as expected.